packages/terra-application-navigation/src/header/_CompactHeader.jsx

```diff
@@ -0,0 +1,165 @@
+   */
+  onSelectSkipToContent: PropTypes.func,
+  /**
+   * Array of navigation items used to determine whether or not the Menu button should render with notifications.
+   */
+  navigationItems: navigationItemsPropType,
+  /**
+   * Key/Value pairs associating a string key entry to a numerical notification count.
+   */
+  notifications: PropTypes.object,
+  /**
+   * @private
+   * The currently active breakpoint.
+   */
+  activeBreakpoint: PropTypes.string,
+  /**
+   * @private
+   * Object containing intl APIs.
+   */
+  intl: intlShape,
+};
+
+const defaultProps = {
+  extensionItems: [],
+  navigationItems: [],
+  notifications: {},
+};
+
+const CompactHeader = ({
+  onSelectMenuButton,
+  titleConfig,
+  extensionItems,
+  onSelectExtensionItem,
+  onSelectSkipToContent,
+  navigationItems,
+  notifications,
+  activeBreakpoint,
+  intl,
+}) => {
+  const previousNotificationsRef = React.useRef();
+  React.useEffect(() => {
+    previousNotificationsRef.current = notifications;
+  });
+
+  function renderMenuButton() {
+    const headerHasCounts = navigationItems.some(item => !!notifications[item.key]);
+    const isPulsed = previousNotificationsRef.current && navigationItems.some((item) => {
+      const previousCount = previousNotificationsRef.current[item.key];
+      const newCount = notifications[item.key];
+      return newCount && (!previousCount || newCount > previousCount);
+    });
+
+    return (
+      <button
+        type="button"
+        className={cx('menu-button')}
+        aria-label={intl.formatMessage({ id: 'Terra.applicationNavigation.header.menuButtonTitle' })}
+        onClick={onSelectMenuButton}
+        onBlur={enableFocusStyles}
+        onMouseDown={disableFocusStyles}
+        data-focus-styles-enabled
+        data-compact-header-toggle
+      >
+        <IconMenu />
+        {headerHasCounts && <ToggleCount value={isPulsed ? 1 : 0} />}
+      </button>
+    );
+  }
+
+  function renderTitle() {
+    if (!titleConfig) {
+      return null;
+    }
+
+    if (titleConfig.element) {
+      return (
+        <div title={titleConfig.title}>
+          {titleConfig.element}
+        </div>
+      );
+    }
+
+    return (
+      <React.Fragment>
+        {titleConfig.headline && <div className={cx('headline')} title={titleConfig.headline}>{titleConfig.headline}</div>}
+        <div className={cx('title')} title={titleConfig.title}>{titleConfig.title}</div>
+        {titleConfig.subline && <div className={cx('subline')} title={titleConfig.subline}>{titleConfig.subline}</div>}
+      </React.Fragment>
+    );
+  }
+
+  function renderExtensions() {
+    if (!extensionItems.length) {
+      return null;
+    }
+
+    return (
+      <Extensions
+        extensionItems={extensionItems}
+        activeBreakpoint={activeBreakpoint}
+        onSelect={onSelectExtensionItem}
+        notifications={notifications}
+      />
+    );
+  }
+
+  return (
+    <div className={cx('compact-header')}>
```

Comment:
Let's update this to the following. You'll also have to change the closing div to a closing header tag.
```suggestion
    <header role="banner" className={cx('compact-header')}>
```

Reply:
Updated.